app/utils/query-sparql-query.js

```diff
@@ -0,0 +1,16 @@
+export async function queryDB(query, sparqlEndpoint = '/sparql') {
```

Comment:
I created this method as it should return the json response of the query. I also saw that [here](https://github.com/lblod/frontend-form-builder/blob/master/app/services/database.js) that there is a database service that does almost the same with the difference that that method returns the full response and not the bindings as json. I could add ... this after my query but that would make it very ugly and not easy to understand.
```
let jsonResponds = await response.json();
this.options =  jsonResponds.results.bindings;
```

Just adding this comment here as we should be aware of this and maybe we can find a solution :)